systemtest/src/test/java/io/strimzi/systemtest/operators/topic/ThrottlingQuotaST.java

```diff
@@ -0,0 +1,357 @@
+                .build();
+        resourceManager.createResource(extensionContext, deleteAdminClientJob.defaultAdmin().build());
+        ClientUtils.waitForClientSuccess(teardownClientName, NAMESPACE, 10);
+        JobUtils.deleteJobWithWait(NAMESPACE, teardownClientName);
+    }
+
+    @ParallelTest
+    void testKafkaAdminTopicOperations(ExtensionContext extensionContext) {
+        final String kafkaUsername = mapWithTestUsers.get(extensionContext.getDisplayName());
+        final String createAdminName = "create-admin-" + mapWithKafkaClientNames.get(extensionContext.getDisplayName());
+        final String deleteAdminName = "delete-admin-" + mapWithKafkaClientNames.get(extensionContext.getDisplayName());
+        final String listAdminName = "list-admin-" + mapWithKafkaClientNames.get(extensionContext.getDisplayName());
+        final String topicNamePrefix = classTopicPrefix + "-simple";
+        int topicsCountBelowQuota = 100;
+
+        setupKafkaUserInNamespace(extensionContext, kafkaUsername);
+        // Create 'topicsCountBelowQuota' topics
+        KafkaAdminClient adminClientJob = new KafkaAdminClient.Builder()
+                .withAdminName(createAdminName)
+                .withBootstrapAddress(KafkaResources.plainBootstrapAddress(CLUSTER_NAME))
+                .withTopicName(topicNamePrefix)
+                .withTopicCount(topicsCountBelowQuota)
+                .withNamespaceName(NAMESPACE)
+                .withTopicOperation(AdminClientOperations.CREATE_TOPICS.toString())
+                .withAdditionalConfig(getAdminClientConfig(kafkaUsername, 240000))
+                .build();
+        resourceManager.createResource(extensionContext, adminClientJob.defaultAdmin().build());
+        ClientUtils.waitForClientSuccess(createAdminName, NAMESPACE, topicsCountBelowQuota);
+        String createPodName = kubeClient().listPods("job-name", createAdminName).get(0).getMetadata().getName();
+        PodUtils.waitUntilMessageIsInPodLogs(createPodName, "All topics created");
+
+        // List 'topicsCountBelowQuota' topics
+        KafkaAdminClient adminClientListJob = new KafkaAdminClient.Builder()
+                .withAdminName(listAdminName)
+                .withBootstrapAddress(KafkaResources.plainBootstrapAddress(CLUSTER_NAME))
+                .withNamespaceName(NAMESPACE)
+                .withTopicOperation(AdminClientOperations.LIST_TOPICS.toString())
+                .withAdditionalConfig(getAdminClientConfig(kafkaUsername, 600000))
+                .build();
+        resourceManager.createResource(extensionContext, adminClientListJob.defaultAdmin().build());
+        ClientUtils.waitForClientSuccess(listAdminName, NAMESPACE, 0);
+        String listPodName = kubeClient().listPods("job-name", listAdminName).get(0).getMetadata().getName();
+        PodUtils.waitUntilMessageIsInPodLogs(listPodName, topicNamePrefix + "-" + (topicsCountBelowQuota - 1));
+        JobUtils.deleteJobWithWait(NAMESPACE, listAdminName);
```

Comment:
Same here, you can create one job and then just change name and operation (or additional config).

Reply:
All is different except bootstrapAddr and namespace, so I would keep those 2 as are .